### PR TITLE
feat(collapsible-sidebar): Add prop to show CollapsibleSidebar collapsed

### PR DESCRIPTION
**Pull request by greathmaster**
### Overview
Currently, if the `CollapsibleSidebar` if the viewport width is greater than `1220px` there is no option to collapse it. This PR adds a prop `narrowItems` to make it collapsible even above `1220px` making the content more visible.

### Current behavior at `1300px`
app hvenkatesh inside-box net_folder_8803997002_s=5j4mo9gvb7mfpvyfjfvj5dxla5ivtjli

### After this PR with `narrowItems = true` at `1300px`
app hvenkatesh inside-box net_folder_8803997002_s=5j4mo9gvb7mfpvyfjfvj5dxla5ivtjli (2)

### CODE REVIEW

**Comment by tjuanitas** [comment]
hey @greathmaster! could you add a screenshot of this change?

File: src/features/collapsible-sidebar/CollapsibleSidebar.js [hunk 1]
```diff
@@ -39,6 +39,9 @@ type Props = {
 
     /** Optionally apply "aria-hidden": "true" to CollapsibleSidebar wrapper */
     isHidden?: boolean,
+
+    /** Whether to show the collapsed version with only icons */
```
Comment (tjuanitas) [inline src/features/collapsible-sidebar/CollapsibleSidebar.js:43]: I'm not familiar with how this is different than `expanded=false`. are `expanded` and `narrowItems` mutually exclusive?

File: src/features/collapsible-sidebar/CollapsibleSidebar.js [hunk 2]
```diff
@@ -39,6 +39,9 @@ type Props = {
...
+    narrowItems?: boolean,
```
Comment (tjuanitas) [inline src/features/collapsible-sidebar/CollapsibleSidebar.js:44]: can prefix booleans with `is`?

File: src/features/collapsible-sidebar/CollapsibleSidebar.js (hunk 1 above)
Comment (greathmaster) [reply src/features/collapsible-sidebar/CollapsibleSidebar.js:43]: In `CollapsibleSidebar.scss` we have a media query that sets the sidebar width to the narrow width (`68px`) when the viewport width is under `1220px`. But above `1220px` it shrinks the sidebar with `expanded=false`, but it leaves the width causing the Files List to be displaced:

app hvenkatesh inside-box net_folder_8803997002_s=5j4mo9gvb7mfpvyfjfvj5dxla5ivtjli (3)

https://github.com/box/box-ui-elements/blob/90e7c476bd7f74c3ad15a49c069f5c5e823f8308/src/features/collapsible-sidebar/CollapsibleSidebar.scss#L115-L137

Question...am I re-creating how `expanded=false` should be working but isn't? ie show the collapsed narrow version regardless of screen size? What is the expected behavior of the `expanded` prop?

**Pull request closed without merging**